doc/source/whatsnew/v1.3.0.rst

```diff
@@ -799,6 +799,7 @@ I/O
 - Bug in :meth:`DataFrame.to_string` misplacing the truncation column when ``index=False`` (:issue:`40907`)
 - Bug in :func:`read_orc` always raising ``AttributeError`` (:issue:`40918`)
 - Bug in the conversion from pyarrow to pandas (e.g. for reading Parquet) with nullable dtypes and a pyarrow array whose data buffer size is not a multiple of dtype size (:issue:`40896`)
+- Bug in :func:`read_excel` loading trailing empty rows/columns for some filetypes (:issue:`41167`)
```

Comment:
can you put netx to this one (or combine as i think they are basically the same)
>  Bug in :func:`read_excel` dropping empty values from single-column spreadsheets (:issue:`39808`)

Reply:
I moved the whatsnew line, but left the two items separate since one bug dropped NaNs within the data and the other bug loaded extra NaNs outside the data. Thanks for reviewing this.